components/esb-tools/plugins/org.wso2.integrationstudio.visualdatamapper.diagram/src/org/wso2/integrationstudio/datamapper/diagram/custom/action/ExportSchemaAction.java

```diff
@@ -74,81 +83,102 @@ public ExportSchemaAction(IWorkbenchPart workbenchPart) {
 		setToolTipText(MENU_TITLE);
 		ISharedImages workbenchImages = PlatformUI.getWorkbench().getSharedImages();
 		setImageDescriptor(workbenchImages.getImageDescriptor(ISharedImages.IMG_TOOL_NEW_WIZARD));
+		this.workbenchPart = workbenchPart;
 	}
 
 	@Override
-	protected void doRun(IProgressMonitor progressMonitor) {
```

Comment:
Seems like the formatting is changed for the complete doRun method. Shall we keep the existing formatting?